index.html

```diff
@@ -209,7 +209,7 @@ <h2 class="project-title hide-for-mobile">Uber Navigation</h2>
                         <li class="tag">css</li>
                         <li class="tag">javaScript</li>
                     </ul>
-                    <button class="button" type="button">See Project</button>
+                    <button class="button see-project" type="button">See Project</button>
```

Comment:
Buttons should be in the center of the project details card. Moreover please add a bit more space between icon in the button and text. Kindly see the below screenshot for reference.
![buttons](https://user-images.githubusercontent.com/35504149/131723967-3eb53cb5-af13-46be-a69f-4328e7bb0d43.png)

Reply:
Please have look at figma template 1 and see the how its style on the page